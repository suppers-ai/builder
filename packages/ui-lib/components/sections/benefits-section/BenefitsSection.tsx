import {
  Accessibility,
  ArrowRight,
  CheckCircle,
  Code2,
  Component,
  Palette,
  Shield,
  Smartphone,
  Target,
  Zap,
  Flower,
} from "lucide-preact";

interface Benefit {
  icon: any;
  title: string;
  description: string;
  features: string[];
  color: string;
}

const benefits: Benefit[] = [
  {
    icon: <Flower size={32} />,
    title: "DaisyUI Components",
    description:
      "Components are built with DaisyUI and are ready to use in your Fresh applications.",
    features: [
      "65+ components included",
      "Perfect API compatibility with Fresh",
      "Regular updates with DaisyUI",
      "No missing features",
    ],
    color: "text-warning",
  },
  {
    icon: <Zap size={32} />,
    title: "Fresh 2.0 Native",
    description:
      "Built specifically for Fresh 2.0 using preact for optimal performance.",
    features: [
      "Server-side rendering",
      "Component islands",
      "Zero runtime overhead",
      "Edge-ready deployment",
    ],
    color: "text-secondary",
  },
  {
    icon: <Code2 size={32} />,
    title: "TypeScript First",
    description: "Fully typed components with IntelliSense support and compile-time safety.",
    features: [
      "100% TypeScript coverage",
      "Auto-completion support",
      "Type-safe props",
      "IDE integration",
    ],
    color: "text-accent",
  },
  {
    icon: <Shield size={32} />,
    title: "Ecosystem Ready",
    description:
      "Components are built to be used in the Suppers ecosystem. This allows for quick development and a consistent user experience across all applications.",
    features: [
      "Secure Authentication",
      "Fast File Storage",
      "Reliable Database",
    ],
    color: "text-success",
  },
];

const features = [
  {
    icon: <Component size={24} />,
    title: "Component Islands",
    description: "Interactive components that hydrate only when needed",
  },
  {
    icon: <Palette size={24} />,
    title: "29 Built-in Themes",
    description: "Switch between themes instantly with built-in theme controller",
  },
  {
    icon: <Smartphone size={24} />,
    title: "Mobile First",
    description: "Responsive design that works perfectly on all devices",
  },
  {
    icon: <Accessibility size={24} />,
    title: "Accessibility",
    description: "WCAG 2.1 compliant with keyboard navigation and screen reader support",
  },
];

export function BenefitsSection() {
  return (
    <section class="py-20 lg:py-32 bg-base-100">
      <div class="px-4 lg:px-6 max-w-6xl mx-auto">
        {/* Section header */}
        <div class="text-center mb-16">
          <h2 class="text-3xl lg:text-4xl font-bold text-base-content mb-4">
            Why Choose Our Component Library?
          </h2>
          <p class="text-lg text-base-content/70 max-w-2xl mx-auto">
            Built by developers, for developers. Get the tools you need to build amazing
            applications with confidence, and harness our ecosystem to speed up your development.
          </p>
        </div>

        {/* Main benefits grid */}
        <div class="grid grid-cols-1 lg:grid-cols-2 gap-8 mb-20">
          {benefits.map((benefit, index) => (
            <div
              key={index}
              class="card bg-base-100 border border-base-300 shadow-lg hover:shadow-xl transition-shadow duration-300"
            >
              <div class="card-body p-8">
                <div class={`${benefit.color} mb-4`}>
                  {benefit.icon}
                </div>
                <h3 class="card-title text-xl mb-3">
                  {benefit.title}
                </h3>
                <p class="text-base-content/70 mb-6">
                  {benefit.description}
                </p>
                <ul class="space-y-2">
                  {benefit.features.map((feature, idx) => (
                    <li key={idx} class="flex items-center gap-3 text-sm">
                      <CheckCircle size={16} className="text-success shrink-0" />
                      <span>{feature}</span>
                    </li>
                  ))}
                </ul>
              </div>
            </div>
          ))}
        </div>

        {/* Additional features */}
        <div class="bg-gradient-to-r from-base-200/50 to-base-300/30 rounded-2xl p-8 lg:p-12">
          <div class="text-center mb-12">
            <h3 class="text-2xl lg:text-3xl font-bold text-base-content mb-4">
              Everything You Need
            </h3>
            <p class="text-base-content/70">
              Comprehensive features that make development a breeze
            </p>
          </div>

          <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-8">
            {features.map((feature, index) => (
              <div key={index} class="text-center">
                <div class="inline-flex items-center justify-center w-12 h-12 bg-primary/10 rounded-lg mb-4">
                  <div class="text-primary">
                    {feature.icon}
                  </div>
                </div>
                <h4 class="font-semibold text-base-content mb-2">
                  {feature.title}
                </h4>
                <p class="text-sm text-base-content/60">
                  {feature.description}
                </p>
              </div>
            ))}
          </div>
        </div>

        {/* CTA */}
        <div class="text-center mt-16">
          <h3 class="text-2xl font-bold text-base-content mb-4">
            Ready to Get Started?
          </h3>
          <p class="text-base-content/70 mb-8 max-w-xl mx-auto">
            Join thousands of developers building amazing applications with our component library.
          </p>
          <div class="flex flex-col sm:flex-row gap-4 justify-center">
            <a
              href="/components"
              class="btn btn-primary btn-lg group"
            >
              Explore Components
              <ArrowRight size={20} className="group-hover:translate-x-1 transition-transform" />
            </a>
            <a
              href="/docs/installation"
              class="btn btn-outline btn-lg"
            >
              Installation Guide
            </a>
          </div>
        </div>
      </div>
    </section>
  );
}
