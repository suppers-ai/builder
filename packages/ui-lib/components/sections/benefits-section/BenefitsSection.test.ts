import { assertEquals, assertStringIncludes } from "https://deno.land/std@0.224.0/assert/mod.ts";
import { assertSnapshot } from "https://deno.land/std@0.224.0/testing/snapshot.ts";
import { renderToString } from "preact-render-to-string";
import { BenefitsSection } from "./BenefitsSection.tsx";

Deno.test("BenefitsSection - basic rendering", () => {
  const html = renderToString(BenefitsSection({}));
  assertStringIncludes(html, "Why Choose Our Component Library?");
  assertStringIncludes(html, "section");
});

Deno.test("BenefitsSection - contains main benefits", () => {
  const html = renderToString(BenefitsSection({}));

  // Check for main benefit titles
  assertStringIncludes(html, "100% DaisyUI Coverage");
  assertStringIncludes(html, "Fresh 2.0 Native");
  assertStringIncludes(html, "TypeScript First");
  assertStringIncludes(html, "Production Ready");
});

Deno.test("BenefitsSection - contains feature descriptions", () => {
  const html = renderToString(BenefitsSection({}));

  // Check for specific feature descriptions
  assertStringIncludes(html, "All 65+ components included");
  assertStringIncludes(html, "Server-side rendering");
  assertStringIncludes(html, "100% TypeScript coverage");
  assertStringIncludes(html, "Extensive test coverage");
});

Deno.test("BenefitsSection - contains additional features", () => {
  const html = renderToString(BenefitsSection({}));

  // Check for additional features section
  assertStringIncludes(html, "Everything You Need");
  assertStringIncludes(html, "Component Islands");
  assertStringIncludes(html, "30+ Themes");
  assertStringIncludes(html, "Mobile First");
  assertStringIncludes(html, "Accessibility");
});

Deno.test("BenefitsSection - contains CTA buttons", () => {
  const html = renderToString(BenefitsSection({}));

  // Check for call-to-action elements
  assertStringIncludes(html, "Ready to Get Started?");
  assertStringIncludes(html, "Explore Components");
  assertStringIncludes(html, "Installation Guide");
});

Deno.test("BenefitsSection - has proper semantic structure", () => {
  const html = renderToString(BenefitsSection({}));

  // Check for semantic HTML structure
  assertStringIncludes(html, "<section");
  assertStringIncludes(html, "card");
  assertStringIncludes(html, "card-body");
  assertStringIncludes(html, "btn");
});

Deno.test("BenefitsSection - includes icon components", () => {
  const html = renderToString(BenefitsSection({}));

  // Check that Lucide icons are rendered (they should create SVG elements)
  assertStringIncludes(html, "<svg");
});

Deno.test("BenefitsSection - has responsive design classes", () => {
  const html = renderToString(BenefitsSection({}));

  // Check for responsive grid and layout classes
  assertStringIncludes(html, "grid");
  assertStringIncludes(html, "lg:");
  assertStringIncludes(html, "max-w-");
});

Deno.test("BenefitsSection - HTML snapshot", async (t) => {
  const html = renderToString(BenefitsSection({}));
  await assertSnapshot(t, html);
});
