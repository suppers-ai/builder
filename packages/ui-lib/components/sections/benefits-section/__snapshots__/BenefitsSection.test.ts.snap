export const snapshot = {};

snapshot[`BenefitsSection - HTML snapshot 1`] = `'<section class="py-20 lg:py-32 bg-base-100"><div class="px-4 lg:px-6 max-w-6xl mx-auto"><div class="text-center mb-16"><h2 class="text-3xl lg:text-4xl font-bold text-base-content mb-4">Why Choose Our Component Library?</h2><p class="text-lg text-base-content/70 max-w-2xl mx-auto">Built by developers, for developers. Get the tools you need to build amazing applications with confidence.</p></div><div class="grid grid-cols-1 lg:grid-cols-2 gap-8 mb-20"><div class="card bg-base-100 border border-base-300 shadow-lg hover:shadow-xl transition-shadow duration-300"><div class="card-body p-8"><div class="text-primary mb-4"><svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-target"><circle cx="12" cy="12" r="10"></circle><circle cx="12" cy="12" r="6"></circle><circle cx="12" cy="12" r="2"></circle></svg></div><h3 class="card-title text-xl mb-3">100% DaisyUI Coverage</h3><p class="text-base-content/70 mb-6">Every single DaisyUI component is implemented and ready to use in your Fresh applications.</p><ul class="space-y-2"><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>All 65+ components included</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Perfect API compatibility</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Regular updates with DaisyUI</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>No missing features</span></li></ul></div></div><div class="card bg-base-100 border border-base-300 shadow-lg hover:shadow-xl transition-shadow duration-300"><div class="card-body p-8"><div class="text-secondary mb-4"><svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-zap"><path d="M4 14a1 1 0 0 1-.78-1.63l9.9-10.2a.5.5 0 0 1 .86.46l-1.92 6.02A1 1 0 0 0 13 10h7a1 1 0 0 1 .78 1.63l-9.9 10.2a.5.5 0 0 1-.86-.46l1.92-6.02A1 1 0 0 0 11 14z"></path></svg></div><h3 class="card-title text-xl mb-3">Fresh 2.0 Native</h3><p class="text-base-content/70 mb-6">Built specifically for Fresh 2.0 with islands architecture and optimal performance.</p><ul class="space-y-2"><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Server-side rendering</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Component islands</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Zero runtime overhead</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Edge-ready deployment</span></li></ul></div></div><div class="card bg-base-100 border border-base-300 shadow-lg hover:shadow-xl transition-shadow duration-300"><div class="card-body p-8"><div class="text-accent mb-4"><svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-code-xml"><path d="m18 16 4-4-4-4"></path><path d="m6 8-4 4 4 4"></path><path d="m14.5 4-5 16"></path></svg></div><h3 class="card-title text-xl mb-3">TypeScript First</h3><p class="text-base-content/70 mb-6">Fully typed components with IntelliSense support and compile-time safety.</p><ul class="space-y-2"><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>100% TypeScript coverage</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Auto-completion support</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Type-safe props</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>IDE integration</span></li></ul></div></div><div class="card bg-base-100 border border-base-300 shadow-lg hover:shadow-xl transition-shadow duration-300"><div class="card-body p-8"><div class="text-success mb-4"><svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-shield"><path d="M20 13c0 5-3.5 7.5-7.66 8.95a1 1 0 0 1-.67-.01C7.5 20.5 4 18 4 13V6a1 1 0 0 1 1-1c2 0 4.5-1.2 6.24-2.72a1.17 1.17 0 0 1 1.52 0C14.51 3.81 17 5 19 5a1 1 0 0 1 1 1z"></path></svg></div><h3 class="card-title text-xl mb-3">Production Ready</h3><p class="text-base-content/70 mb-6">Battle-tested components with comprehensive testing, accessibility compliance, and detailed documentation.</p><ul class="space-y-2"><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Extensive test coverage</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>WCAG 2.1 AA compliant</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Performance optimized</span></li><li class="flex items-center gap-3 text-sm"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-check-big"><path d="M21.801 10A10 10 0 1 1 17 3.335"></path><path d="m9 11 3 3L22 4"></path></svg><span>Cross-browser compatible</span></li></ul></div></div></div><div class="bg-gradient-to-r from-base-200/50 to-base-300/30 rounded-2xl p-8 lg:p-12"><div class="text-center mb-12"><h3 class="text-2xl lg:text-3xl font-bold text-base-content mb-4">Everything You Need</h3><p class="text-base-content/70">Comprehensive features that make development a breeze</p></div><div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-8"><div class="text-center"><div class="inline-flex items-center justify-center w-12 h-12 bg-primary/10 rounded-lg mb-4"><div class="text-primary"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-component"><path d="M5.5 8.5 9 12l-3.5 3.5L2 12l3.5-3.5Z"></path><path d="m12 2 3.5 3.5L12 9 8.5 5.5 12 2Z"></path><path d="M18.5 8.5 22 12l-3.5 3.5L15 12l3.5-3.5Z"></path><path d="m12 15 3.5 3.5L12 22l-3.5-3.5L12 15Z"></path></svg></div></div><h4 class="font-semibold text-base-content mb-2">Component Islands</h4><p class="text-sm text-base-content/60">Interactive components that hydrate only when needed</p></div><div class="text-center"><div class="inline-flex items-center justify-center w-12 h-12 bg-primary/10 rounded-lg mb-4"><div class="text-primary"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-palette"><circle cx="13.5" cy="6.5" r=".5" fill="currentColor"></circle><circle cx="17.5" cy="10.5" r=".5" fill="currentColor"></circle><circle cx="8.5" cy="7.5" r=".5" fill="currentColor"></circle><circle cx="6.5" cy="12.5" r=".5" fill="currentColor"></circle><path d="M12 2C6.5 2 2 6.5 2 12s4.5 10 10 10c.926 0 1.648-.746 1.648-1.688 0-.437-.18-.835-.437-1.125-.29-.289-.438-.652-.438-1.125a1.64 1.64 0 0 1 1.668-1.668h1.996c3.051 0 5.555-2.503 5.555-5.554C21.965 6.012 17.461 2 12 2z"></path></svg></div></div><h4 class="font-semibold text-base-content mb-2">29 Built-in Themes</h4><p class="text-sm text-base-content/60">Switch between themes instantly with built-in theme controller</p></div><div class="text-center"><div class="inline-flex items-center justify-center w-12 h-12 bg-primary/10 rounded-lg mb-4"><div class="text-primary"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-smartphone"><rect width="14" height="20" x="5" y="2" rx="2" ry="2"></rect><path d="M12 18h.01"></path></svg></div></div><h4 class="font-semibold text-base-content mb-2">Mobile First</h4><p class="text-sm text-base-content/60">Responsive design that works perfectly on all devices</p></div><div class="text-center"><div class="inline-flex items-center justify-center w-12 h-12 bg-primary/10 rounded-lg mb-4"><div class="text-primary"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-accessibility"><circle cx="16" cy="4" r="1"></circle><path d="m18 19 1-7-6 1"></path><path d="m5 8 3-3 5.5 3-2.36 3.5"></path><path d="M4.24 14.5a5 5 0 0 0 6.88 6"></path><path d="M13.76 17.5a5 5 0 0 0-6.88-6"></path></svg></div></div><h4 class="font-semibold text-base-content mb-2">Accessibility</h4><p class="text-sm text-base-content/60">WCAG 2.1 compliant with keyboard navigation and screen reader support</p></div></div></div><div class="text-center mt-16"><h3 class="text-2xl font-bold text-base-content mb-4">Ready to Get Started?</h3><p class="text-base-content/70 mb-8 max-w-xl mx-auto">Join thousands of developers building amazing applications with our component library.</p><div class="flex flex-col sm:flex-row gap-4 justify-center"><a href="/components" class="btn btn-primary btn-lg group">Explore Components<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-arrow-right"><path d="M5 12h14"></path><path d="m12 5 7 7-7 7"></path></svg></a><a href="/docs/installation" class="btn btn-outline btn-lg">Installation Guide</a></div></div></div></section>'`;
